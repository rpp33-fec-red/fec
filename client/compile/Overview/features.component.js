import React from 'react';

import PropTypes from 'prop-types';

function Features(props){
  return props.features.map(function(feature,index) {
    return <li key={index}> <span> {feature.feature}</span><span> {feature.value}</span></li>;
  });
}
Features.propTypes = {
  features:PropTypes.array
};
export default Features;