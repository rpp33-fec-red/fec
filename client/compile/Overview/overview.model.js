import React from 'react';
class Model extends React.Component {
  constructor(props) {
    super(props)
    this.props = props;
    this.state = {

    }

  }









}
export default Model;