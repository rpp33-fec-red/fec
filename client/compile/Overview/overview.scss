.overview{
  width:100%;
  height:800px;
  background:orange;
  display:flex;
  flex-wrap:nowrap;
}