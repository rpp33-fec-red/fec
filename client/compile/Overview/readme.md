<h1 align="center"> The overview wigit</h1>

<p align="center">
  <img src="./overview.png" alt="overviewimg" width="800px" height="600px"/>
  <br>
  <i>Overview is a wiget where a customer can overview the product details. the most important details of them all.
  </i>
  <br>
</p>

## Documentation
Getting started with how to use this wigits code;


### API MODEL

#how to import the model;

```javascript
# this model connects to the backend to make requests for data
import Model from 'pathtomodelFile';
var newModel = new Model(false|true);
# if you choose false then you are not ready to deploy and the functionality may be slightly diffrent.

```

### STATE



