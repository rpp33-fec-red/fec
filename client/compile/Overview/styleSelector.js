
import React from 'react';
import PropTypes from 'prop-types';

class StyleSelector extends React.Component {
//picks a certain style and shows all images for that style
//if props.showImages
//if props.showStyles

  constructor(props){
    super(props);

    this.state = {
      imageSelected:null,
      styleSelected:null,
      expanded:false,
      expandedStyles:{'transform':'rotate(0deg)'}
    };
    this.expand = this.expand.bind(this);
    this.showStyles = this.showStyles.bind(this);
  }




  showStyles(){
    var that = this;
    function clickchangeStyle(){
      that.props.changeStyle(this.index);
    };

    return (
      <div className="grid-ct">
        { this.props.styles.map(function(style,index){
          return <div className="style"  onClick={clickchangeStyle.bind({index:index})} key={style.style_id}> <img src={style.photos[0].thumbnail_url}></img></div>;
        })
        }
      </div>);
  }

  expand(){
    var newexpanded = !this.state.expanded;
    this.setState({expanded:newexpanded}, () => {
      var answer;
      if (this.state.expanded){
        answer = {'transform':'rotate(270deg)'};
      } else {
        answer = {'transform':'rotate(0deg)'};
      }
      this.setState({expandedStyles: answer },()=>{
        console.log(this.state);
      });
    });
  }



  showImages() {
    var that = this;

    function Modal(){
      return that.props.photos.map((photo, index)=>{
        return (<div className="box" key={index}>
          <img onClick={that.props.imageClick.bind({index:index})} src={photo.thumbnail_url} ></img>
        </div>);
      });

    }

    return (
      <div className="imageSelector">
        <div className="box" onClick={this.expand}> <svg className="arrow" style={this.state.expandedStyles} xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0z" fill="none"/><path d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"/></svg></div>
        { this.state.expanded ? <Modal /> : null }


      </div>);
  }

  render(){
    if (this.props.showImages !== undefined){
      return this.showImages();
    }
    if (this.props.showStyles !== undefined){
      return this.showStyles();
    }
  }



}

StyleSelector.propTypes = {
  url: PropTypes.any,
};

export default StyleSelector;