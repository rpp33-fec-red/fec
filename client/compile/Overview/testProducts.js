var  testData =
{
  "id": 64620,
  "campus": "hr-rpp",
  "name": "Camo Onesie",
  "slogan": "Blend in to your crowd",
  "description": "The So Fatigues will wake you up and fit you in. This high energy camo will have you blending in to even the wildest surroundings.",
  "category": "Jackets",
  "default_price": "140.00",
  "created_at": "2022-01-28T00:20:03.466Z",
  "updated_at": "2022-01-28T00:20:03.466Z",
  "styles": [
    {
      "style_id": 398226,
      "name": "White Sole",
      "original_price": "120.00",
      "sale_price": null,
      "default?": false,
      "photos": [
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1561861422-a549073e547a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1561861422-a549073e547a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
        },
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1560072810-1cffb09faf0f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1560072810-1cffb09faf0f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
        }
      ],
      "skus": {
        "2313270": {
          "quantity": 14,
          "size": "7"
        },
        "2313271": {
          "quantity": 25,
          "size": "7.5"
        },
        "2313272": {
          "quantity": 9,
          "size": "8"
        },
        "2313273": {
          "quantity": 2,
          "size": "8.5"
        },
        "2313274": {
          "quantity": 18,
          "size": "9"
        },
        "2313275": {
          "quantity": 12,
          "size": "9.5"
        },
        "2313276": {
          "quantity": 10,
          "size": "10"
        },
        "2313277": {
          "quantity": 18,
          "size": "10.5"
        },
        "2313278": {
          "quantity": 11,
          "size": "11"
        },
        "2313279": {
          "quantity": 35,
          "size": "11.5"
        },
        "2313280": {
          "quantity": 25,
          "size": "12"
        }
      }
    },
    {
      "style_id": 398227,
      "name": "Black Sole",
      "original_price": "120.00",
      "sale_price": null,
      "default?": false,
      "photos": [
        {
          "thumbnail_url": "https://images.unsplash.com/uploads/1412198532414025532c0/6a31309c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/uploads/1412198532414025532c0/6a31309c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1600&q=80"
        },
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1454024183771-42d54053cd75?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1454024183771-42d54053cd75?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
        }
      ],
      "skus": {
        "2313281": {
          "quantity": 14,
          "size": "7"
        },
        "2313282": {
          "quantity": 25,
          "size": "7.5"
        },
        "2313283": {
          "quantity": 9,
          "size": "8"
        },
        "2313284": {
          "quantity": 2,
          "size": "8.5"
        },
        "2313285": {
          "quantity": 18,
          "size": "9"
        },
        "2313286": {
          "quantity": 12,
          "size": "9.5"
        },
        "2313287": {
          "quantity": 10,
          "size": "10"
        },
        "2313288": {
          "quantity": 18,
          "size": "10.5"
        },
        "2313289": {
          "quantity": 11,
          "size": "11"
        },
        "2313290": {
          "quantity": 35,
          "size": "11.5"
        },
        "2313291": {
          "quantity": 25,
          "size": "12"
        }
      }
    },
    {
      "style_id": 398228,
      "name": "Tan Sole",
      "original_price": "120.00",
      "sale_price": null,
      "default?": false,
      "photos": [
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1465124982537-9f918f1e1aaf?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1465124982537-9f918f1e1aaf?ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80"
        },
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1449505278894-297fdb3edbc1?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1449505278894-297fdb3edbc1?ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80"
        }
      ],
      "skus": {
        "2313292": {
          "quantity": 14,
          "size": "7"
        },
        "2313293": {
          "quantity": 25,
          "size": "7.5"
        },
        "2313294": {
          "quantity": 9,
          "size": "8"
        },
        "2313295": {
          "quantity": 2,
          "size": "8.5"
        },
        "2313296": {
          "quantity": 18,
          "size": "9"
        },
        "2313297": {
          "quantity": 12,
          "size": "9.5"
        },
        "2313298": {
          "quantity": 10,
          "size": "10"
        },
        "2313299": {
          "quantity": 18,
          "size": "10.5"
        },
        "2313300": {
          "quantity": 11,
          "size": "11"
        },
        "2313301": {
          "quantity": 35,
          "size": "11.5"
        },
        "2313302": {
          "quantity": 25,
          "size": "12"
        }
      }
    },
    {
      "style_id": 398229,
      "name": "Red Sole",
      "original_price": "120.00",
      "sale_price": null,
      "default?": false,
      "photos": [
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1454177643390-7f100d1bbeec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1454177643390-7f100d1bbeec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80"
        }
      ],
      "skus": {
        "2313303": {
          "quantity": 14,
          "size": "7"
        },
        "2313304": {
          "quantity": 25,
          "size": "7.5"
        },
        "2313305": {
          "quantity": 9,
          "size": "8"
        },
        "2313306": {
          "quantity": 2,
          "size": "8.5"
        },
        "2313307": {
          "quantity": 18,
          "size": "9"
        },
        "2313308": {
          "quantity": 12,
          "size": "9.5"
        },
        "2313309": {
          "quantity": 10,
          "size": "10"
        },
        "2313310": {
          "quantity": 18,
          "size": "10.5"
        },
        "2313311": {
          "quantity": 11,
          "size": "11"
        },
        "2313312": {
          "quantity": 35,
          "size": "11.5"
        },
        "2313313": {
          "quantity": 25,
          "size": "12"
        }
      }
    },
    {
      "style_id": 398230,
      "name": "Yellow Sole",
      "original_price": "120.00",
      "sale_price": null,
      "default?": false,
      "photos": [
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1499013819532-e4ff41b00669?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1499013819532-e4ff41b00669?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1100&q=80"
        },
        {
          "thumbnail_url": "https://images.unsplash.com/photo-1459631836437-1c2824882e3f?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80",
          "url": "https://images.unsplash.com/photo-1459631836437-1c2824882e3f?ixlib=rb-1.2.1&auto=format&fit=crop&w=1650&q=80"
        }
      ],
      "skus": {
        "2313314": {
          "quantity": 14,
          "size": "7"
        },
        "2313315": {
          "quantity": 25,
          "size": "7.5"
        },
        "2313316": {
          "quantity": 9,
          "size": "8"
        },
        "2313317": {
          "quantity": 2,
          "size": "8.5"
        },
        "2313318": {
          "quantity": 18,
          "size": "9"
        },
        "2313319": {
          "quantity": 12,
          "size": "9.5"
        },
        "2313320": {
          "quantity": 10,
          "size": "10"
        },
        "2313321": {
          "quantity": 18,
          "size": "10.5"
        },
        "2313322": {
          "quantity": 11,
          "size": "11"
        },
        "2313323": {
          "quantity": 35,
          "size": "11.5"
        },
        "2313324": {
          "quantity": 25,
          "size": "12"
        }
      }
    }
  ]

};


export default testData;