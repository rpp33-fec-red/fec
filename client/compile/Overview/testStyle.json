{
  "product": {
      "style_id": 398195,
      "name": "Forest Green & Black",
      "original_price": "140.00",
      "sale_price": null,
      "default?": true,
      "photos": [
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
          },
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1534011546717-407bced4d25c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2734&q=80"
          },
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1549831243-a69a0b3d39e0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1549831243-a69a0b3d39e0?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2775&q=80"
          },
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1527522883525-97119bfce82d?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1527522883525-97119bfce82d?ixlib=rb-1.2.1&auto=format&fit=crop&w=668&q=80"
          },
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1556648202-80e751c133da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1556648202-80e751c133da?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
          },
          {
              "thumbnail_url": "https://images.unsplash.com/photo-1532543491484-63e29b3c1f5d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80",
              "url": "https://images.unsplash.com/photo-1532543491484-63e29b3c1f5d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80"
          }
      ],
      "skus": {
          "2313078": {
              "quantity": 8,
              "size": "XS"
          },
          "2313079": {
              "quantity": 16,
              "size": "S"
          },
          "2313080": {
              "quantity": 17,
              "size": "M"
          },
          "2313081": {
              "quantity": 10,
              "size": "L"
          },
          "2313082": {
              "quantity": 15,
              "size": "XL"
          },
          "2313083": {
              "quantity": 4,
              "size": "XL"
          }
      }
  }
}