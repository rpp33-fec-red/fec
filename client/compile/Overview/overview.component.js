import React from 'react';
import LeftContainer from './leftContainer/leftcontainer.component.js';
import RightContainer from './rightContainer/rightcontainer.component.js';
import './overview.scss';
import PropTypes from 'prop-types';
import OverviewModel from './overview.model.js';
import Features from './features.component.js';


class Overview extends OverviewModel {

  constructor(props) {
    super(props);
    this.Core = this.Core.bind(this);
  }

  componentDidMount(){
    this.getReviews();
    this.getProductData();
  }

  Core(){
    return (
      <React.Fragment>
        <div className="overview" style={{ backgroundImage:'url(./beach.jpeg)' }}>
          <LeftContainer clickImage={this.clickImage} moveUp={this.moveUp} moveDown={this.moveDown} image={ this.state.image} ThumbnailIndex={this.state.ThumbnailIndex} thumbArray={this.state.thumbArray} imageClick={this.imageClick}/>
          <RightContainer ratings={this.state.ratings} reviews={this.state.reviews.length} changeStyle={this.changeStyle} styles={this.state.product.styles} productInfo={this.state.product.styles[0]}  />
        </div>
        <p>{this.state.product.description}</p>
        <ul>
          { Array.isArray(this.state.product.features) ?  <Features features={this.state.product.features}></Features>: null }
        </ul>
      </React.Fragment>
    );
  }
  render(){
    return this.getComponent(this.Core);
  }
}

Overview.propTypes = {
  model:PropTypes.any
};
export default Overview;