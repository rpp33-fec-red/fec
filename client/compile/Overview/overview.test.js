/* eslint-disable no-undef */
import renderer from 'react-test-renderer';
import React from 'react';

import Overview from './overview.component.js';
import Model from '../model.js';
var model = new Model(false);
//get other components and inject the props with fake data

// eslint-disable-next-line no-undef
describe('Overview component should render',function(){

  var getData = function (){
    return model.getData;
  };

  var mainTree = renderer.create(<Overview getProducts={getData}/>).toJSON();
  test('should have overview component in browser',function(){
    expect(mainTree).toMatchSnapshot();
  });

  test('should have child with div', function(){
    expect(mainTree.children[0].type).toEqual('div');
  });
});

describe('should have correct data',function() {
  var getData = function () {
    return model.getData;
  };
  var mainTree = renderer.create(<Overview getProducts={getData}/>);
  var instance =mainTree.getInstance();
  console.log(instance)
  test('should have props onProduct',function(){
    expect(instance.state['onProduct']).not.toBe(undefined);
  });

  test('should have props style',function(){
    expect(instance.state['onStyle']).not.toBe(undefined);
  });
});



describe('data should get passed down to left container then to image container',function() {
  var getData = function () {
    return model.getData;
  };
  var mainTree = renderer.create(<Overview getProducts={getData}/>);
  var instance =mainTree.getInstance();
  console.log(instance);
  test('instance props of overview should have function getdData from model',function(){
    expect(typeof instance.props.getProducts).toBe('function');
  });

  test('instance props of overview should have function getdData from model',function(){
    expect(typeof instance.props.getProducts).toBe('function');
  })
  var children = instance._reactInternals.return;
  console.log(children)
  test('CHILD has class left-container',function(){
    expect(children[0]).not.toBe(undefined);
  })

});