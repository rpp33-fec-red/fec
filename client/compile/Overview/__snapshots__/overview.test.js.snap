// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview component should render should have overview component in browser 1`] = `
Array [
  <div
    className="overview"
  >
    <div
      className="leftCt"
    >
      <div
        className="image-holder"
      >
        <img
          src="https://images.unsplash.com/photo-1561861422-a549073e547a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=668&q=80"
        />
        <div
          className="hoverModal"
          onClick={[Function]}
          style={
            Object {
              "background": "transparent",
              "position": "absolute",
            }
          }
        />
        <div
          className="imageSelector"
        >
          <div
            className="box"
            onClick={[Function]}
          >
             
            <svg
              className="arrow"
              fill="#000000"
              height="24px"
              style={
                Object {
                  "transform": "rotate(0deg)",
                }
              }
              viewBox="0 0 24 24"
              width="24px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
    <div
      className="rightCt"
    >
      <div
        className="main-content"
      >
        <div
          className="reviewWrapper"
        >
          <a
            onClick={[Function]}
          >
            readall reviews
          </a>
        </div>
        <span
          className="mediumText"
        >
          Catagory:
          Jackets
        </span>
        <div
          className="title"
        >
           
          Camo Onesie
        </div>
        <span
          className="price"
        >
          $ 
          120.00
        </span>
      </div>
      <div
        className="style-selectors"
      >
        <div
          className="grid-ct"
        >
          <div
            className="style"
            onClick={[Function]}
            style={
              Object {
                "border": "2px solid red",
                "borderRadius": "55px",
              }
            }
          >
             
            <svg
              fill="#000000"
              height="30px"
              style={
                Object {
                  "position": "absolute",
                  "right": "-5px",
                  "top": "-5px",
                }
              }
              viewBox="0 0 24 24"
              width="30px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0V0z"
                fill="none"
              />
              <path
                d="M18 7l-1.41-1.41-6.34 6.34 1.41 1.41L18 7zm4.24-1.41L11.66 16.17 7.48 12l-1.41 1.41L11.66 19l12-12-1.42-1.41zM.41 13.41L6 19l1.41-1.41L1.83 12 .41 13.41z"
              />
            </svg>
            <img
              src="https://images.unsplash.com/photo-1561861422-a549073e547a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
            />
          </div>
          <div
            className="style"
            onClick={[Function]}
            style={
              Object {
                "border": "2px solid transparent",
              }
            }
          >
             
            <img
              src="https://images.unsplash.com/uploads/1412198532414025532c0/6a31309c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
            />
          </div>
          <div
            className="style"
            onClick={[Function]}
            style={
              Object {
                "border": "2px solid transparent",
              }
            }
          >
             
            <img
              src="https://images.unsplash.com/photo-1465124982537-9f918f1e1aaf?ixlib=rb-1.2.1&auto=format&fit=crop&w=300&q=80"
            />
          </div>
          <div
            className="style"
            onClick={[Function]}
            style={
              Object {
                "border": "2px solid transparent",
              }
            }
          >
             
            <img
              src="https://images.unsplash.com/photo-1454177643390-7f100d1bbeec?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
            />
          </div>
          <div
            className="style"
            onClick={[Function]}
            style={
              Object {
                "border": "2px solid transparent",
              }
            }
          >
             
            <img
              src="https://images.unsplash.com/photo-1499013819532-e4ff41b00669?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80"
            />
          </div>
        </div>
      </div>
      <div
        className="option-selectors"
      >
        <div
          className="selector"
        >
          <label
            htmlFor="styles"
          >
            Choose a style_id:
          </label>
          <select
            className="drop-down sku-dropdown"
            id="styles"
            name="styles"
          >
            <option>
              Select size
            </option>
            <option
              onClick={[Function]}
            >
              7
            </option>
            <option
              onClick={[Function]}
            >
              7.5
            </option>
            <option
              onClick={[Function]}
            >
              8
            </option>
            <option
              onClick={[Function]}
            >
              8.5
            </option>
            <option
              onClick={[Function]}
            >
              9
            </option>
            <option
              onClick={[Function]}
            >
              9.5
            </option>
            <option
              onClick={[Function]}
            >
              10
            </option>
            <option
              onClick={[Function]}
            >
              10.5
            </option>
            <option
              onClick={[Function]}
            >
              11
            </option>
            <option
              onClick={[Function]}
            >
              11.5
            </option>
            <option
              onClick={[Function]}
            >
              12
            </option>
          </select>
          <select
            className="drop-down"
            id="qty"
            name="qty"
          >
            <option
              onClick={[Function]}
            >
              1
            </option>
            <option
              onClick={[Function]}
            >
              2
            </option>
            <option
              onClick={[Function]}
            >
              3
            </option>
            <option
              onClick={[Function]}
            >
              4
            </option>
            <option
              onClick={[Function]}
            >
              5
            </option>
            <option
              onClick={[Function]}
            >
              6
            </option>
            <option
              onClick={[Function]}
            >
              7
            </option>
            <option
              onClick={[Function]}
            >
              8
            </option>
            <option
              onClick={[Function]}
            >
              9
            </option>
            <option
              onClick={[Function]}
            >
              10
            </option>
            <option
              onClick={[Function]}
            >
              11
            </option>
            <option
              onClick={[Function]}
            >
              12
            </option>
            <option
              onClick={[Function]}
            >
              13
            </option>
          </select>
        </div>
        <div
          className="selector"
        >
          <button
            className="addToCart"
          >
            Add to Cart
          </button>
          <button
            className="addFavorite"
          >
            star
          </button>
        </div>
      </div>
    </div>
  </div>,
  <p>
    The So Fatigues will wake you up and fit you in. This high energy camo will have you blending in to even the wildest surroundings.
  </p>,
  <ul />,
]
`;
